tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml

```diff
@@ -0,0 +1,78 @@
+[setting.synthStataUsdcMaxMarketCollateral]
+defaultValue = "<%= parseEther(String(5_000_000)) %>"
+
+[setting.spotStrictStalenessTolerance]
+defaultValue = "60"
+
+[setting.sstatausdc_issuance_ratio]
+defaultValue = "<%= MaxUint256 %>"
+
+[setting.sstatausdc_liquidation_ratio]
+defaultValue = "<%= parseEther('1.1') %>"
+
+[setting.sstatausdc_liquidation_reward]
+defaultValue = "<%= parseEther('0.005') %>"
+
+[setting.sstatausdc_min_delegation]
+defaultValue = "<%= parseEther('100') %>"
+
+[invoke.SpotMarketProxy_createSynth_stataUSDC]
```

Comment:
We need an e2e test for this spot market. Especially considering the use of a special oracle.

Reply:
Added a draft test. Unsure how best to simulate the `aUSDC` rebase.

Reply:
Looks perfectly good, e2e here are more of a smoke tests to ensure "normal" usage scenario is working. Any special cases, reverts, etc are meant to be covered by integration/unit tests in synthetix-v3. 

Thanks for taking the time and getting the test set up and running.